Misc/NEWS.d/next/Library/2025-05-19-18-12-42.gh-issue-88994.7avvVu.rst

```diff
@@ -0,0 +1,2 @@
+Change :func:`datetime.datetime.now` to half-even rounding for
+consistency with :func:`datetime.datetime.fromtimestamp`.
```

Comment:
If you want you can also add "Patch by (Your name however you want it rendered)." here.